backend/app/api/services/thread_service.py

```diff
@@ -0,0 +1,87 @@
+import uuid
+from typing import Optional
+
+from app.api.schemas import Filter, Message, MessageCreate, SupportCreate, SupportThread
+
+_threads: list[SupportThread] = []
+
+
+class SupportThreadService:
+    """
+    class for manipulation SupportThread entities.
+    Note: some functions should return exactly pydantic model!
+    """
+
+    @staticmethod
+    async def create(thread: SupportCreate) -> SupportThread:
+        """from function expected that it will store the thread in db"""
+        thread.id = uuid.uuid4().int
+        thread_new = SupportThread(id=thread.id, question=thread.question)
+        _threads.append(thread_new)
+
+        return thread_new
+
+    @staticmethod
+    async def find_all(flt: Optional[Filter] = None) -> list[SupportThread]:
+        """from function expected to return all thread entities from db"""
+        if flt and flt == Filter.free:
```

Comment:
Why not just `flt == Filter.free`? Is there a convention or a technical reason for that?

(same at backend/app/api/services/volunteer_service.py)